pkg/client/client.go

```diff
@@ -992,6 +992,18 @@ func GetPrimaryProxy(url string) (string, error) {
 	return smap.ProxySI.DirectURL, nil
 }
 
+func UnregisterTarget(url, sid string) error {
+	err := HTTPRequest("DELETE", url+"/"+dfc.Rversion+"/"+dfc.Rcluster+"/"+"daemon"+"/"+sid, nil)
+	// FIXME: do something better than sleep
+	time.Sleep(time.Second * 3)
+	return err
+}
+
+func RegisterTarget(sid string, smap *dfc.Smap) error {
```

Comment:
Since the function does not check anything, it can be turned into one-liner:
return HTTPRequest("POST", smap.Tmap[sid].DirectURL+"/"+dfc.Rversion+"/"+dfc.Rdaemon+"/"+dfc.Rregister, nil)
In this case, it is not clear why we need it in the library